packages/cf-component-toggle/src/ToggleTheme.js

```diff
@@ -0,0 +1,46 @@
+export default baseTheme => ({
+  display: 'block',
+  position: 'relative',
+  margin: 0,
+  height: '2.26667rem',
+  width: '5.334rem',
+  borderRadius: '2px',
+  borderWidth: '1px',
+  borderStyle: 'solid',
+  borderColor: '#aaa',
```

Comment:
I don't wanna wait forever for something to happen here so I'm gonna merge this with my two approvals.

If someone wants to make a issue and assign it to me with what they would like to happen here I will try to schedule it to be worked on as soon as possible.